libraries/AP_Notify/Display_SH1106_I2C.h

```diff
@@ -31,7 +32,7 @@ class Display_SH1106_I2C: public Display_Backend {
     void _timer();
 
     AP_HAL::OwnPtr<AP_HAL::Device> _dev;
-    uint8_t _displaybuffer[SH1106_COLUMNS * SH1106_ROWS_PER_PAGE];
+    uint8_t _displaybuffer[SH1106_COLUMNS * SH1106_PAGES];
```

Comment:
This just looks wrong; your display buffer size is rows*columns - not columns * pages.

Similarly... just about everywhere else.

Reply:
I am going to be honest here, I just wanted to make my first contribution and tried to find the easiest issue. I pretty much did exactly what was laid out by the person who started the issue, #7065.

Reply:
OK, thanks.  I'll need to go back and study that.

FWIW, your implementation here is fine - I'm just not sure its the right thing to do!

Apologies for that bad label - feel free to grab another GoodFirstIssue - if you want to pass it by someone grab us on our Discord server.

Reply:
@charlie-mtz OK, so I went back and read that issue.

This is the critical bit from there: "or vice versa, depending on what is liked better" - and I assure you I rather prefer calculating pages instead.

Are you interested in inverting this logic?